app/src/androidTest/java/com/FindFaceTest.java

```diff
@@ -86,7 +86,13 @@ public void performTest() {
         Context appContext = InstrumentationRegistry.getInstrumentation().getTargetContext();
         // Name of model does not have any impact on results, it is never used.
         model = GlobalData.getModel(appContext,
-                appContext.getResources().getString(R.string.model_Facenet)).neuralModel;
+                GlobalData.getUserSettings(appContext).getString(
```

Comment:
Wouldn't putting getUserSettings and getResources() results in variables be more convenient and clear?

SharedPreferences userSettings = GlobalData.getUserSettings(appContext);
Resources res = appContext.getResources();
etc.

Reply:
I agree that it would probably be shorter than what it is

Reply:
i disagree, android studio make it shorter to string value, but i will do yours way